API/AddContact.php

```diff
@@ -0,0 +1,48 @@
+<?php
+	$inData = getRequestInfo();
+	
+    $firstName = $inData["firstName"];
+    $lastName = $inData["lastName"];
+    $email = $inData["email"];
+    $phone = $inData["phone"];
+
```

Comment:
Each contact should know what user it belongs to, hence we need to store $userID
```suggestion
$userID = $inData["userID"];
```